### ISSUE TITLE
Incorrect `class` and `aria-current` attribute output in "Plain HTML" markup style

### ISSUE THREAD

**Issue by vicobot-0815** [x3p0-dev/x3p0-breadcrumbs#18]
Hi!

First off, thanks for this fabulous plugin - it works like a charm!

I’ve noticed a small issue when selecting the **"Plain HTML" markup style** for the block. The generated `<li>` elements have malformed `class` attributes, where the closing quote appears before the post type or context is appended. Additionally, the `aria-current` attribute on the last breadcrumb item is broken.

### Steps to Reproduce

1. Spin up a fresh site with only the X3P0 Breadcrumbs plugin installed and activated. 
2. Create a page with child pages and a post assigned to a category.
3. In the Site Editor, add the X3P0 Breadcrumbs block to the 'Pages' and 'Single Posts' templates.
4. Select 'Plain HTML' as the Markup Style for the block and save the templates.
5. View the page or post in the frontend.
6. Look at the HTML of the block in the page source.

### Expected Behavior  

- Each `<li>` should have a correctly formatted class attribute, e.g., `class="breadcrumbs__crumb breadcrumbs__crumb--home"`.
- The last breadcrumb should have a valid aria-current attribute, e.g., `aria-current="page"`.

### Actual Behavior  
The class attributes for `<li>` come out as
```html
<li class="breadcrumbs__crumb breadcrumbs__crumb--"home>
<li class="breadcrumbs__crumb breadcrumbs__crumb--"post>
<li class="breadcrumbs__crumb breadcrumbs__crumb--"term>
```

The `aria-current` attribute for the last `<li>` comes out as 
`<li class="breadcrumbs__crumb breadcrumbs__crumb-- aria-current="page""post>`.

### Environment  
I tested with a fresh site on  [https://playground.wordpress.net](https://playground.wordpress.net) (sorry, I don't know how to create a blueprint link) so i got its defaults: 
- Wordpress 6.8
- TT5 theme
- PHP 8.3
- deleted all other plugins
- installed and activated only the latest version of the X3P0 Breadcrumbs plugin
- tested in Firefox 143

Thanks & greetings

**Comment by justintadlock** [comment]
Fixed in: https://github.com/x3p0-dev/x3p0-breadcrumbs/commit/2eee5c338d44c5880951ec1d535e9d66b53960fd
